documentation/meta/media_properties/frontend.md

```diff
@@ -0,0 +1,126 @@
+# Frontend Media Properties
+
+_This document is auto-generated from the source code in
+[/frontend/scripts/document-media.js](https://github.com/WordPress/openverse/blob/main/frontend/scripts/document-media.js)._
+
+This is a list of the media properties, with the descriptions of corresponding
+TypeScript types that are used in the frontend to hold media objects.
+
+## General notes
+
+The columns are sorted alphabetically. Both interfaces `ImageDetail` and
+`AudioDetail` inherit from interface `Media` so the tables for those interfaces
+only show their additional fields.
+
+Some types in the "Type" column may refer to interfaces or types defined in the
+frontend codebase. In such cases it is advised to refer to the source file
+[/frontend/src/types/media.ts](https://github.com/WordPress/openverse/blob/main/frontend/src/types/media.ts)
+and trace the source of the type definitions. Such types are italicised for
+clarity.
+
+## Interface `Media`
+
+| Name                  | Type                   | Optional? |
+| --------------------- | ---------------------- | --------- |
+| `attribution`         | `string`               |           |
+| `category`            | `string \| null`       |           |
+| `creator`             | `string`               | ✓         |
+| `creator_url`         | `string`               | ✓         |
+| `description`         | `string`               | ✓         |
+| `detail_url`          | `string`               |           |
+| `fields_matched`      | `string[]`             | ✓         |
+| `filesize`            | `string`               | ✓         |
+| `filetype`            | `string`               | ✓         |
+| `foreign_landing_url` | `string`               |           |
+| `frontendMediaType`   | _`SupportedMediaType`_ |           |
+| `id`                  | `string`               |           |
+| `isSensitive`         | `boolean`              |           |
+| `license`             | _`License`_            |           |
+| `license_url`         | `string`               | ✓         |
+| `license_version`     | _`LicenseVersion`_     |           |
+| `originalTitle`       | `string`               |           |
+| `provider`            | `string`               |           |
+| `providerName`        | `string`               | ✓         |
+| `related_url`         | `string`               |           |
+| `sensitivity`         | _`Sensitivity[]`_      |           |
+| `source`              | `string`               | ✓         |
+| `sourceName`          | `string`               | ✓         |
+| `tags`                | _`Tag[]`_              |           |
+| `thumbnail`           | `string`               | ✓         |
+| `title`               | `string`               |           |
+| `url`                 | `string`               |           |
+
+### id
+
+the UUID4 identifier of the media item
```

Comment:
Idk how possible/reasonable it is but this might be worth sanitizing so we capitalize the first letter and add a period on the end if it doesn't exist already!

Reply:
These are not sentences so a capital first letter and trailing space didn't feel right. They're more similar to dictionary entries and are formatted similarly to those.